www/js/app.js

```diff
@@ -792,6 +804,9 @@ define(['jquery', 'zimArchiveLoader', 'util', 'uiUtil', 'cookies','abstractFiles
     // Since late 2014, all ZIM files should use relative URLs
     var regexpImageUrl = /^(?:\.\.\/|\/)+(I\/.*)$/;
     var regexpMetadataUrl = /^(?:\.\.\/|\/)+(-\/.*)$/;
+    
+    // Cache for CSS styles contained in ZIM [kiwix-js #335] 
```

Comment:
@mossroy Are you in favour of this kind of reference in comments to a discussion in the repository for future devs? Alternatively, I could put:

`// Cache for CSS styles contained in ZIM (significantly speeds up subsequent page display)`

Reply:
I don't have a strong opinion on that.
It's probably a good idea, as it might help the reader to have more information. But I think it should not be put *instead* of a relevant comment.
In this case, it might be a mix of both like : 

```javascript
// Cache for CSS styles contained in ZIM.
// It significantly speeds up subsequent page display. See kiwix-js issue #335
```

By the way, is the bracket syntax `[kiwix-js #335]` a way to make it automatically recognized by some tools?

Reply:
The bracket syntax was just a form of abbreviation, not a tool thing.
Will use your suggestion.